java/compaction/compaction-status-store/src/test/java/sleeper/compaction/status/store/job/StoreCompactionJobUpdatesIT.java

```diff
@@ -58,6 +58,26 @@ public void shouldReportCompactionJobStartedSeparatelyFromCreation() {
                 .containsExactly(startedStatusWithDefaults(job));
     }
 
+    @Test
+    public void shouldReportCompactionJobUncommittedSeparatelyFromOthers() {
+        // Given
+        Partition partition = singlePartition();
+        FileReferenceFactory fileFactory = fileFactory(partition);
+        CompactionJob job = jobFactory.createCompactionJob(
+                List.of(fileFactory.rootFile(100L)),
+                partition.getId());
+
+        // When
+        store.jobCreated(job);
+        store.jobStarted(compactionJobStarted(job, defaultStartTime()).taskId(DEFAULT_TASK_ID).build());
+        store.jobFinished(compactionJobFinished(job, defaultSummary()).taskId(DEFAULT_TASK_ID).build());
+
+        // Then
+        assertThat(getAllJobStatuses())
+                .usingRecursiveFieldByFieldElementComparator(IGNORE_UPDATE_TIMES)
+                .containsExactly(finishedUncommittedStatusWithDefaults(job));
+    }
+
     @Test
     public void shouldReportCompactionJobFinishedSeparatelyFromOthers() {
```

Comment:
This test seems to be an exact duplicate of shouldReportCompactionJobCommittedSeparately.